KubeArmor/build/kubearmor-test-containerd.yaml

```diff
@@ -37,13 +37,50 @@ spec:
       labels:
         kubearmor-app: kubearmor
     spec:
+      serviceAccountName: kubearmor
+      nodeSelector:
+        kubernetes.io/os: linux
+      tolerations:
+      - operator: Exists
+      hostPID: true
+      hostNetwork: true
+      restartPolicy: Always
+      dnsPolicy: ClusterFirstWithHostNet
+      initContainers:
+      - name: init
+        image: kubearmor/kubearmor-init
+        imagePullPolicy: Never
+        securityContext:
+          privileged: true
+        volumeMounts:
+          - mountPath: /opt/kubearmor/BPF
+            name: bpf
+          - mountPath: /lib/modules
+            name: lib-modules-path
+            readOnly: true
+          - mountPath: /sys/fs/bpf
+            name: sys-fs-bpf-path
+          - mountPath: /sys/kernel/security
+            name: sys-kernel-security-path
+          - mountPath: /sys/kernel/debug
+            name: sys-kernel-debug-path
+          - mountPath: /media/root/etc/os-release
+            name: os-release-path
+            readOnly: true
+          - mountPath: /usr/src
```

Comment:
we will face problems with such `mountPath` .. gke COS latest rapid releases do not have `/usr/src` folder.
Ref: #648 

Is this yaml only for test purpose?

Reply:
These YAML is only for test purposes. We will have to modify `deployments/get` with the new stuff for end user deployments and kArmor.

Adding that to TODO.